guide/admin/tags.md

```diff
@@ -0,0 +1,57 @@
+## User Handbook
+### Creating, editing and deleting Tags
+
+Managing your forum tags requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Tags** to reveal the configuration page to the right hand side.
+
+![Tags Screenshot](687474703a2f2f692e696d6775722e636f6d2f354f58595454642e706e67.png)
+
+### Creating a new Tag
+
+A tag is a way of categorising posts made by a user, they are primarily set by the staff of the forum (admins).
+
+On the configuration page for **Tags** you'll need to click **Create Tag** which will bring up the create tag box:
+
+![SS Tag Creation Box](687474703a2f2f692e696d6775722e636f6d2f304772355739742e706e67.png)
+
+### Create Tag Components
+
+#### Name:
+
+The name is what you'd like the tag to be called, also known as the category title.
+
+#### Slug:
+
+The slug is a string that goes into the URL, essentially it's a version of the name of the tag that is all lower-case, and uses dashes instead of spaces.
+
+Unless you need to, you generally don't need to do anything with this, as when you type in the **Name** the slug auto-fills in.
+
+#### Description:
+
+The description of the tag will appear alongside the **name** in the tag selection post when making a post, it's a short few words giving a more in-depth description of the category it belongs to.
+
+#### Colour:
+
+This is the colour you would like the tag to appear in on the posts, and forums, the colour box uses **HTML Colour Codes** beginning with a **#** you can Google around for HTML Colour Codes, there are plenty of websites on this.
+
+Once you've filled in the appropriate information in the Create Tag prompt, click on **Save Changes** to add the new tag.
+
+### Moving the Tag between Primary, and Secondary tags
+
+By default, new tags created will be **Secondary Tags** also known as **Sub Categories**, sometimes you'll want these to be primary tags, to do this, click and drag the tag to the **Primary** slot at the top, follow the below animation to see how this is done:
+
+![Anim - Moving Tags](687474703a2f2f692e696d6775722e636f6d2f765738796e42732e676966.gif)
```

Comment:
I added it to the creating tag section, and added a screenshot showing the result.